OpenBGPD.conf

```diff
@@ -26,17 +26,36 @@ group "blackholes" {
   multihop 63
 
   neighbor $projectton {
-    remote-as 65618
+    remote-as 55016
     descr "Project TON"
   }
 }
 
 ## rules section
 deny quick to group blackholes
-match from group blackholes set {
-  nexthop blackhole
-  }
-allow quick from group blackholes
+match from group blackholes set { nexthop blackhole }
+deny from group blackholes
+
+# 65535:666 = Blackhole
+# This will allow any prefixes with the BLACKHOLE community set:
+allow from group blackholes community BLACKHOLE
+
+# 65332:888 = BOGONs
+# This will allow any prefixes that are BOGONs:
+allow from group blackholes community 65332:888
```

Comment:
Why are we allowing BOGONs?

Reply:
I've (hopefully) added some clarity to the comments to describe what is going on.  Please re-re-review when you have a moment.